src/features/farming/crops/components/landExpansion/Plot.tsx

```diff
@@ -176,7 +176,7 @@ export const Plot: React.FC<Props> = ({
     >
       <div className="absolute bottom-1 w-full h-full">
         <Soil
-          className="absolute bottom-0"
+          className="absolute bottom-0 pointer-events-none"
```

Comment:
This change prevents large crops from blocking the hover on the crop behind.

![flowers](https://user-images.githubusercontent.com/41215134/184060674-8c7a1831-740c-46a8-9a13-e82a4fbd0097.png)